frontend/src/component/Activity/Preview.jsx

```diff
@@ -0,0 +1,239 @@
+import React from 'react'
+import { translate } from 'react-i18next'
+import { Link } from 'react-router-dom'
+import {
+  buildFilePreviewUrl as jpgPreviewUrl,
+  removeExtensionOfFilename,
+  removeInteractiveContentFromHTML,
+  CONTENT_TYPE,
+  SCREEN_SIZE
+} from 'tracim_frontend_lib'
+import { FETCH_CONFIG, PAGE } from '../../util/helper.js'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import { getHTMLPreview } from '../../action-creator.async.js'
+
+require('./Preview.styl')
+
+// INFO - RJ - 2020-11-17 - possible preview sizes with their media queries, ordered from smallest to biggest
+const PREVIEW_WIDTHS = [
+  [
+    `(max-width: ${SCREEN_SIZE.MAX_SM})`,
+    376 // 400px - margins (10 * 2px) - borders (2 * 1px)
+  ],
+  [
+    '',
+    776 // 800px - margins (10 * 2px) - borders (2 * 1px)
+  ]
+]
+
+const MAX_PREVIEW_HEIGHT = 300
+
+class Preview extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      previewComponent: this.isHtmlPreview() ? null : this.getJPEGPreview(),
+      previewOverflow: false,
+      previewLoading: this.isHtmlPreview(),
+      previewUnavailable: false
+    }
+  }
+
+  isHtmlPreview () {
+    const type = this.props.content.content_type
+    return type === CONTENT_TYPE.HTML_DOCUMENT || type === CONTENT_TYPE.THREAD
+  }
+
+
+  handleUnavailablePreview = () => {
+    this.setState({
+      previewLoading: false,
+      previewComponent: null,
+      previewUnavailable: true
+    })
+  }
+
+  setLoadingPreview () {
+    this.setState({ previewLoading: true })
+  }
+
+
```

Comment:
```suggestion

```

Reply:
okay, I did some linting, sorry for that.